### PR TITLE
DOC: How to cite PyPDF2

### LINKED ISSUE #1475: Citation information for PyPDF2
## Explanation

Hello, I was wondering how I could cite PyPDF2 in my research paper. I used the software to convert part of a corpus for an NLP project.  BibTex formatting would be preferred! Thanks!

### PR DESCRIPTION
**Pull request by MartinThoma**
Closes #1475

### CODE REVIEW

**Comment by MartinThoma** [comment]
@maphew @pubpub-zz @mstamy2 @exiledkingcc What do you think about this guide how to cite PyPDF2?

Examples for other software:

* [Numpy](https://numpy.org/citing-numpy/): A lot of people are mentioned, but by far not all. They wrote a white-paper
* [Scipy](https://scipy.org/citing-scipy/): They mention "SciPy 1.0 Contributors"
* [Tensorflow](https://www.tensorflow.org/about/bib): A whitepaper on Zenodo
* [Lasagne](https://github.com/Lasagne/Lasagne/wiki/Lasagne-Citation-(BibTeX))

**Comment by pubpub-zz** [comment]
perhaps add also a link to the github contributors page to have a dynamic list of the contributors

**Comment by MartinThoma** [comment]
@pubpub-zz I've replaced the URL by the contributors page URL

> and be sure to not forget anyone

The contributors page is for sure missing a lot of people. I will add the once which I perceive as the most important ones (the people who I have added as authors into bibtex directly - but even there I'm not sure if I might have forgotten people)

File: docs/meta/faq.md [hunk 1]
```diff
@@ -33,3 +33,21 @@ Projects that depend on PyPDF2:
...
+@misc{pypdf2,
+ title         = {The {PyPDF2} library},
+ author        = {Mathieu Fenniak and
+                  Matthew Stamy and
+                  pubpub-zz and
+                  Martin Thoma and
+                  Matthew Peveler and
+                  exiledkingcc and {PyPDF2 Contributors}},
+ year          = {2022},
+ url           = {https://pypdf2.readthedocs.io/en/latest/meta/CONTRIBUTORS.html}
```
Comment (MasterOdin) [inline docs/meta/faq.md:51]: While I get the sentiment behind using this URL, and it is nice, I personally would find this strange as the URL compared to most every other software I've cited. I would recommend having this URL point at the actual project URL, and then secondarily use the [`note`](https://www.bibtex.com/f/note-field/) field to point here:

```suggestion
url           = {https://pypi.org/project/PyPDF2/}
note        = {See https://pypdf2.readthedocs.io/en/latest/meta/CONTRIBUTORS.html for all contributors}
```

**Review by MasterOdin — COMMENTED** [review]
My recommendation that if you wanted to make PyPDF2 citable, you should add it to zenodo. Having a permanent DOI that one can use to reference back to a work is nice for a number of reasons in my experience.

**Comment by MartinThoma** [comment]
> you should add it to zenodo

I think that's a good idea. I'll do that once we merged back into `pypdf`

**Comment by MartinThoma** [comment]
I'll merge this now. Feel free to add comments here / open issues. I'm open to adjust this :-)

**Pull request merged**